proposals/flux.md

```diff
@@ -0,0 +1,182 @@
+## Project name and description
+
+"Weave Flux" (or just "Flux") is the GitOps Kubernetes Operator: it
+automatically ensures that the state of a cluster matches the config
+in git and images are upgraded automatically. Flux uses an operator in
+the cluster to trigger deployments inside Kubernetes. It supports Helm
+as well, with a subproject called the "Helm operator".
+
+Weaveworks started the project in July 2016 and is receiving more and
+more contributions from a growing community. The team of maintainers
+is very active and made up from people from different companies.
+
+## Alignment with CNCF charter mission
+
+TODO
+
+## Sponsor from TOC
+
+Michelle Noorali, Xiang Li.
```

Comment:
The project still has some couplings with weavecloud (https://github.com/weaveworks/flux/search?q=weave+cloud&unscoped_q=weave+cloud). I hope we can decouple flux with that, and make the project more vendor neural before entering the sandbox.

Reply:
Yes, I assume it's a prerequisite for any hard-coded hostnames etc., to be excised, and help text and other documentation to be at least reworded to not prefer (or exclusively mention) Weave Cloud. A small amount of the documentation is _only_ relevant to Weave Cloud and can just be removed.

I would also propose moving the "upstream" component (which has the Weave Cloud specific connection code) into another repo, to be deployed as a sidecar. We can discuss other measures, too.

Reply:
We've created a milestone for decoupling Flux from Weave Cloud: https://github.com/weaveworks/flux/milestone/16
It's being actively worked on.